### DISCUSSION TITLE
Sending event with Kubebuilder v3

### DISCUSSION THREAD

**Discussion by fjammes** [kubernetes-sigs/kubebuilder#2465] [Support] [answered]
The [Kubebuilder V3 documentation](https://book.kubebuilder.io/) explains that it talks about  "How to batch multiple events into a single reconciliation call". However I could not find any information about event management in this documentation.

Could you please help about this?

**Comment by fjammes** [comment]
Add this question in stackoverflow: https://stackoverflow.com/questions/70529847/how-to-send-events-with-kubebuilder-v3-operator-sdk

**Comment by camilamacedo86** [comment, marked as answer]
Hi @fjammes, 

The answer to your question is tracked in the issue https://github.com/kubernetes-sigs/kubebuilder/issues/2897 for we are able to doc it. So, I am marking this one as answered.

**Comment by camilamacedo86** [comment]
Closing since it is answered.
